assets/scss/base/_icons.scss

```diff
@@ -1,3 +1,18 @@
+/*
+Arrow
+
+The arrow icon is used to represent an open or closed child element.
+
+Markup:
+<i class="arrow {{modifier_class}}"></i>
+
+.arrow--right   - Arrow pointing to the right
+.arrow--left    - Arrow pointing to the left
+.arrow--up      - Arrow pointing up
+.arrow--down    - Arrow pointing down
+
+Styleguide Icons.arrow
```

Comment:
Shouldn't this be an uppercase 'a', so `Icons.Arrow` to be consistent with others?